pandas/tests/test_categorical.py

```diff
@@ -1013,7 +1009,7 @@ def test_tab_complete_warning(self, ip):
         pytest.importorskip('IPython', minversion="6.0.0")
         from IPython.core.completer import provisionalcompleter
 
-        code = "import pandas as pd; c = pd.Categorical([])"
+        code = "import pandas as pd; c = Categorical([])"
```

Comment:
@jschendel @jreback why is removing pd OK here?  We have a couple tests with this pattern that look fishy to me, but they are passing on master so im confused